perception/lidar_centerpoint/src/node.cpp

```diff
@@ -39,23 +39,23 @@ LidarCenterPointNode::LidarCenterPointNode(const rclcpp::NodeOptions & node_opti
 : Node("lidar_center_point", node_options), tf_buffer_(this->get_clock())
 {
   const float score_threshold =
-    static_cast<float>(this->declare_parameter<double>("score_threshold", 0.35));
+    static_cast<float>(this->declare_parameter<double>("score_threshold"));
   const float circle_nms_dist_threshold =
     static_cast<float>(this->declare_parameter<double>("circle_nms_dist_threshold"));
   const auto yaw_norm_thresholds =
     this->declare_parameter<std::vector<double>>("yaw_norm_thresholds");
   const std::string densification_world_frame_id =
-    this->declare_parameter("densification_world_frame_id", "map");
+    this->declare_parameter<std::string>("densification_world_frame_id");
   const int densification_num_past_frames =
-    this->declare_parameter("densification_num_past_frames", 1);
-  const std::string trt_precision = this->declare_parameter("trt_precision", "fp16");
+    this->declare_parameter<int>("densification_num_past_frames");
+  const std::string trt_precision = this->declare_parameter<std::string>("trt_precision");
   const std::string encoder_onnx_path = this->declare_parameter<std::string>("encoder_onnx_path");
   const std::string encoder_engine_path =
     this->declare_parameter<std::string>("encoder_engine_path");
   const std::string head_onnx_path = this->declare_parameter<std::string>("head_onnx_path");
   const std::string head_engine_path = this->declare_parameter<std::string>("head_engine_path");
   class_names_ = this->declare_parameter<std::vector<std::string>>("class_names");
-  has_twist_ = this->declare_parameter("has_twist", false);
+  has_twist_ = this->declare_parameter<bool>("has_twist");
```

Comment:
`has_twist` has a different in pointpainting. Sould not they be uniform?

Reply:
Fixed at https://github.com/autowarefoundation/autoware.universe/pull/6591/commits/f30631b30a14a03c426ce063e311c084db23a97a